test/performance/observed_concurency_test.go

```diff
@@ -82,64 +155,58 @@ func TestObservedConcurrency(t *testing.T) {
 	test.CleanupOnInterrupt(func() { TearDown(perfClients, logger, names) }, logger)
 
 	logger.Info("Creating a new Service")
-	svc, err := test.CreateLatestService(logger, clients, names)
+	objs, err := test.CreateRunLatestServiceReady(logger, clients, &names, &test.Options{ContainerConcurrency: 1})
```

Comment:
Haven't you encounter any issues related with #2579?

Reply:
No, this isn't exactly huge load though